src/js/elements/mapchip.js

```diff
@@ -74,13 +74,30 @@ export class MapChip extends Component {
 
     getDefaultFilters = (args) => {
         let defaultFilters = [];
+        let defaultFilter = {};
 
         if (typeof args.data.chartConfiguration.filter !== 'undefined') {
             args.data.chartConfiguration.filter['defaults'].forEach((f) => {
-                let defaultFilter = {
-                    group: f.name,
-                    value: f.value,
-                    default: true
+
+                let defaultExists = args.data.data.filter(function (o) {
+                  return o[f.name]==f.value;
+                }).length > 0;
+
+                if (defaultExists) {
+                    defaultFilter = {
+                        group: f.name,
+                        value: f.value,
+                        default: true
+                    }
+                } else {
+                    let existingGroup = Object.keys(args.data.data[0])[0]
```

Comment:
`Object.keys(args.data.data[0])` results to ["age", "race", "count", "gender", "grade 9"]

Are we certain that the order here is always maintained so that we can pick the first item?

Reply:
Instead, we can pick the filter defaults name, `f.name`

Reply:
The modified code will look like:
```
let existingGroup = f.name
let existingValue = args.data.data[0][f.name]

defaultFilter = {
                        group: existingGroup,
                        value: existingValue,
                        default: true
                    }
```

Reply:
@waracci Thanks, I don't really like using the first item either - what if no valid filters exist. The problem with f.name is if an admin sets it to an invalid filter group e.g. 'age group' instead of 'age'. My thought that if an admin sets an invalid filter, detect this and default to the first valid one

Reply:
Do we have a case where we can have no valid filters?

Reply:
I think so, for example the SANEF profile. Though there is a very small chance of an admin adding a filter if none exist